src/sources/tokenList.ts

```diff
@@ -37,7 +40,7 @@ function buildTokenListSource({ db, url, tag, logger }: Dependencies) {
 
       return {
         networkId: chain.id,
-        address: token.address.toUpperCase(),
+        address: checksumAddress(token.address as `0x${string}`),
```

Comment:
Can't we use viem's `getAddress` here?

Reply:
be my guest
https://github.com/l2beat/token-db/pull/31/commits/3fdd751b81be81e4bab5ac233e5c5e79bfc039d3